x/uibc/gmp/types.go

```diff
@@ -15,8 +15,8 @@ const (
 	TypeSendToken
 )
 
-// Message is attached in ICS20 packet memo field
-type Message struct {
+// GMPMemo is attached in ICS20 packet memo field
+type GMPMemo struct {
```

Comment:
```suggestion
type Memo struct {
```
GMP is already a package name